notebooks/tutorials/beginner_workflow/footprint_workflow.ipynb

```diff
@@ -25,7 +25,7 @@
     "import os\n",
     "import matplotlib.pyplot as plt\n",
     "from openghg.localclient import get_obs_surface\n",
-    "from openghg.modules import Emissions, FOOTPRINTS, ObsSurface\n",
+    "from openghg.modules import Emissions, Footprints, ObsSurface\n",
```

Comment:
Does the import statement here need to be updated to import from somewhere other than `openghg.modules`? Is it `openghg.store` now?

Reply:
Good catch, I'll change them now.